### PR TITLE
[TACHYON-1233] Write unit tests for UnderFileSystemCluster

### PR DESCRIPTION
**Pull request by aaudiber**
https://tachyon.atlassian.net/browse/TACHYON-1233

This addresses comments from https://github.com/amplab/tachyon/pull/2325

### CODE REVIEW

File: minicluster/src/test/java/tachyon/underfs/UnderFileSystemClusterTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+  /**
+   * Tests that the {UnderFileSystemCluster{@link #readEOFReturnsNegativeTest()} method will return
+   * true only when the cluster type is "tachyon.underfs.hdfs.LocalMiniDFSCluster".
+   */
+  @Test
+  public void readEOFReturnsNegativeTest() {
+    Whitebox.setInternalState(UnderFileSystemCluster.class, "sUnderFSClass",
+            (String) null);
+    boolean resultFlag = UnderFileSystemCluster.readEOFReturnsNegative();
```
Comment (yupeng9) [inline minicluster/src/test/java/tachyon/underfs/UnderFileSystemClusterTest.java]: variable `resultFlag` seems not necessary.

Comment (aaudiber) [inline minicluster/src/test/java/tachyon/underfs/UnderFileSystemClusterTest.java]: Agreed, I've removed it

**Comment by aaudiber** [comment]
Jenkins, test this please.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**